docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,765 @@
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Fixing externals paths...
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Running externals post install...
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Creating /home/diracuser/DIRAC/bashrc
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Defaults written to defaults-DIRAC.cfg
+  2019-04-11 14:47:02 UTC dirac-install [NOTICE]  Executing /home/diracuser/DIRAC/scripts/dirac-externals-requirements...
+  2019-04-11 14:47:03 UTC dirac-install [NOTICE]  DIRAC properly installed
+
+You will notice that among other things, the installation created a `~/DIRAC/bashrc` file. This file must be sourced whenever you want to use dirac client.
+
+In principle, your system administrator will have managed the CA for you. In this specific case, since we have our own CA, we will just link the client installation CA with the server one::
+
+  mkdir -p ~/DIRAC/etc/grid-security/
+  ln -s /opt/dirac/etc/grid-security/certificates/ ~/DIRAC/etc/grid-security/certificates
+
+The last step is to configure the client to talk to the proper configuration service. This is easily done by creating a `~/.dirac.cfg` file with the following content::
+
+  DIRAC
+  {
+    Setup = MyDIRAC-Production
+    Configuration
+    {
+      Servers = dips://dirac-tuto:9135/Configuration/Server
+    }
+  }
+
+You should now be able to get a proxy::
+
+  [diracuser@dirac-tuto DIRAC]$ source ~/DIRAC/bashrc
+  [diracuser@dirac-tuto DIRAC]$ dirac-proxy-init
+  Generating proxy...
+  Proxy generated:
+  subject      : /C=ch/O=DIRAC/OU=DIRAC CI/CN=ciuser/emailAddress=lhcb-dirac-ci@cern.ch/CN=460648814
+  issuer       : /C=ch/O=DIRAC/OU=DIRAC CI/CN=ciuser/emailAddress=lhcb-dirac-ci@cern.ch
+  identity     : /C=ch/O=DIRAC/OU=DIRAC CI/CN=ciuser/emailAddress=lhcb-dirac-ci@cern.ch
+  timeleft     : 23:59:59
+  DIRAC group  : dirac_user
+  rfc          : True
+  path         : /tmp/x509up_u501
+  username     : ciuser
+  properties   : NormalUser
+
+
+And you can observe that the Configuration Service has served the client::
+
+  [diracuser@dirac-tuto DIRAC]$ grep ciuser /opt/dirac/runit/Configuration/Server/log/current
+  2019-04-11 14:54:10 UTC Configuration/Server NOTICE: Executing action ([::1]:33394)[dirac_user:ciuser] RPC/getCompressedDataIfNewer(<masked>)
+  2019-04-11 14:54:10 UTC Configuration/Server NOTICE: Returning response ([::1]:33394)[dirac_user:ciuser] (0.00 secs) OK
+
+--------------
+Use the WebApp
+--------------
+
+This section is to be executed as `diracuser`.
+
+First you need to convert your user certificate into a `p12` format (you will be prompt for a password, you can leave it empty)::
+
+  cd ~/.globus/
+  openssl pkcs12 -export -out certificate.p12 -inkey userkey.pem -in usercert.pem
```

Comment:
There's a `dirac-cert-convert.sh` script for this.

Reply:
That does the opposite: from X509 to p12